tests/Controller/EnMarche/CommitteeControllerTest.php

```diff
@@ -285,6 +288,87 @@ public function testAuthenticatedCommitteeHostCanShowCommitteePage()
+    {
+        $this->authenticateAsAdherent($this->client, 'jacques.picard@en-marche.fr');
+        $crawler = $this->client->request('GET', '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
+
+        $this->assertSame(10, $result->count());
+    }
+
+    public function testDisplayEditLinkWithNormaleUser()
+    {
+        $this->authenticateAsAdherent($this->client, 'francis.brioul@yahoo.com');
+        $crawler = $this->client->request('GET', '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
+
+        $this->assertSame(5, $result->count());
+    }
+
+    public function testEditMessage()
+    {
+        /** @var EntityManagerInterface $em */
+        $em = $this->get('doctrine')->getManager();
+        $committee = $em->getRepository(Committee::class)->findOneBy(['slug' => 'en-marche-paris-8']);
+        $messages = $em->getRepository(CommitteeFeedItem::class)->findMostRecentFeedEvent($committee->getUuid());
+        $this->authenticateAsAdherent($this->client, 'jacques.picard@en-marche.fr');
+
+        $crawler = $this->client->request('GET', '/comites/en-marche-paris-8/timeline/'.$messages->getId().'/edit');
+        $this->isSuccessful($this->client->getResponse());
+        $form = $crawler->selectButton('committee_feed_message_send')->form();
+        $this->assertSame($messages->getContent(), $form->get('committee_feed_message[content]')->getValue());
+        $form->setValues(['committee_feed_message[content]' => $messages->getContent().' test']);
+        $this->client->submit($form);
+        $this->assertClientIsRedirectedTo('/comites/en-marche-paris-8', $this->client);
+        $this->client->followRedirect();
+        self::assertContains($messages->getContent().' test', $this->client->getResponse()->getContent());
+    }
+
+    public function testDeleteMessage()
+    {
+        /** @var EntityManagerInterface $em */
+        $em = $this->get('doctrine')->getManager();
+        $committee = $em->getRepository(Committee::class)->findOneBy(['slug' => 'en-marche-paris-8']);
+        $messages = $em->getRepository(CommitteeFeedItem::class)->findMostRecentFeedEvent($committee->getUuid());
+        $this->authenticateAsAdherent($this->client, 'jacques.picard@en-marche.fr');
+
+        $this->client->request('GET', '/comites/en-marche-paris-8/timeline/'.$messages->getId().'/delete');
+        $this->assertClientIsRedirectedTo('/comites/en-marche-paris-8', $this->client);
+        $this->client->followRedirect();
+        self::assertNotContains($messages->getContent(), $this->client->getResponse()->getContent());
+    }
+
+    public function testDeleteEditDenied()
+    {
+        /** @var EntityManagerInterface $em */
+        $em = $this->get('doctrine')->getManager();
```

Comment:
you can retrieve the manager with `$this->manager` thanks to the `init()` method